holidays/countries/switzerland.py

```
@@ -725,16 +730,32 @@ def _populate_subdiv_zh_public_holidays(self):
         # Whit Monday.
         self._add_whit_monday(tr("Pfingstmontag"))
 
+        # Saint Stephen's Day.
+        self._add_christmas_day_two(tr("Stephanstag"))
+
+    def _populate_subdiv_zh_de_facto_holidays(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_populate_subdiv_zh_de_facto_holidays`

Add return type annotation: `None`

(ANN202)